03-nlp/languages.http

```diff
@@ -0,0 +1,255 @@
+      }
+    ]
+  }
+}
+
+### Sentiment Analysis
+POST {{ language endpoint }}/language/:analyze-text?api-version=2023-04-01 HTTP/1.1
+Content-Type: application/json
+Ocp-Apim-Subscription-Key: {{ Language-Key }}
+{
+  "kind": "SentimentAnalysis",
+  "parameters": {
+    "modelVersion": "latest"
+  },
+  "analysisInput": {
+    "documents": [
+      {
+        "id": "1",
+        "language": "en",
+        "text": "Great atmosphere. Close to plenty of restaurants, hotels, and transit! Staff are friendly and helpful."
+      }
+    ]
+  }
+}
+
+### Key Phrase Extraction
+POST {{ language endpoint }}/language/:analyze-text?api-version=2023-04-01 HTTP/1.1
+Host: 67121-demo.cognitiveservices.azure.com
+Content-Type: application/json
+Ocp-Apim-Subscription-Key: {{ Language-Key }}
+
+{
+  "kind": "KeyPhraseExtraction",
+  "parameters": {
+    "modelVersion": "latest"
+  },
+  "analysisInput": {
+    "documents": [
+      {
+        "id": "1",
+        "language": "en",
+        "text": "Microsoft was founded by Bill Gates and Paul Allen."
+      },
+      {
+        "id": "2",
+        "language": "en",
+        "text": "Text Analytics is one of the Azure Cognitive Services."
+      },
+      {
+        "id": "3",
+        "language": "en",
+        "text": "My cat might need to see a veterinarian."
+      }
+    ]
+  }
+}
+
+### Named Entity Recognition (NER)
+POST {{ language endpoint }}/language/:analyze-text?api-version=2023-04-01 HTTP/1.1
+Host: 67121-demo.cognitiveservices.azure.com
+Content-Type: application/json
+Ocp-Apim-Subscription-Key: {{ Language-Key }}
+
+{
+  "kind": "EntityRecognition",
+  "parameters": {
+    "modelVersion": "latest"
+  },
+  "analysisInput": {
+    "documents": [
+      {
+        "id": "1",
+        "language": "en",
+        "text": "Microsoft was founded by Bill Gates and Paul Allen."
+      },
+      {
+        "id": "2",
+        "language": "en",
+        "text": "Pike place market is my favorite Seattle attraction."
+      }
+    ]
+  }
+}
+
+### PII Detection
+POST {{ language endpoint }}/language/:analyze-text?api-version=2023-04-01 HTTP/1.1
+Host: 67121-demo.cognitiveservices.azure.com
+Content-Type: application/json
+Ocp-Apim-Subscription-Key: {{ Language-Key }}
+
+{
+  "kind": "PiiEntityRecognition",
+  "parameters": {
+    "modelVersion": "latest"
+  },
+  "analysisInput": {
+    "documents": [
+      {
+        "id": "1",
+        "language": "en",
+        "text": "My SSN is 859-98-0987"
+      },
+      {
+        "id": "2",
+        "language": "en",
+        "text": "Your ABA number - 111000025 - is the first 9 digits in the lower left hand corner of your personal check."
+      },
+      {
+        "id": "3",
+        "language": "en",
+        "text": "Is 998.214.865-68 your Brazilian CPF number?"
+      }
+    ]
+  }
+}
+
+### Conversation Summarisation
+POST {{ language endpoint }}/language/analyze-conversations/jobs?api-version=2023-04-15-preview HTTP/1.1
```

Comment:
- Comments in `.http` files should only have one `#`. The three `###` are how the REST Client extension separates REST requests. One `#` is for comments.
- Please use camelCase for the variables, so the language endpoint should be `languageEndpoint` which is consistent to the others we have in the repo.